rb/selenium-webdriver.gemspec

```diff
@@ -48,6 +48,7 @@ Gem::Specification.new do |s|
   s.require_paths = ['lib']
 
   s.add_dependency 'base64', ['~> 0.2']
+  s.add_dependency 'curb', ['~> 1.0.5']
```

Comment:
That was by design, the idea is that people can add `curb` gem to their own Gemfile if they want to use this HTTP client.

Reply:
Oh great thank you for the explanation :)

Reply:
For the purpose of this PR, it would make sense to add `curb` as a development dependency so you could run specs for it.

Also, once Gemfile.lock is updated, please run `bazel build @bundle` to regenerate the checksums. The command should print `gem_checksums = {...}` to the output which you can then copy/paste to MODULE.bazel.